### ISSUE TITLE
Error while building SwinjectStoryBoard with XCode 12.5

### ISSUE THREAD

**Issue by shalinipk** [Swinject/SwinjectStoryboard#154]
Anyone ran into this build error ?

```
SwinjectStoryboard/Sources/iOS-tvOS/UIStoryboard+Swizzling.m:55:13: error: cast to smaller integer type 'BOOL' (aka 'signed char') from 'id' [-Werror,-Wpointer-to-int-cast]
        if ((BOOL)[[SwinjectStoryboard class] performSelector:@selector(isCreatingStoryboardReference)]) {
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

I ended up manually removing the BOOL type casting so that carthage builds.

My cartfile entry looks like this -  github "Swinject/SwinjectStoryboard" "0ca45c83a8aa398c153d8a036c95abb4343cfa0c"

Looking forward to version 2.2.1 for SwinjectStoryboard. Thanks!!

**Comment by tweetjay** [comment]
Also looking forward! :)

**Comment by Bawenang** [comment]
Is there any news about this fix? I see that there are two pull reqs that are trying to fix this bug.

**Comment by yoichitgy** [comment]
Merged PR #155 and closing this issue. Thanks!
